### PR TITLE
commited

### PR DESCRIPTION
**Pull request by Byon2**
Okay finished with the test programs and commented out the print statements from my first functions

### CODE REVIEW

**Comment by kasgel** [comment]
@Byon2 
You have a file called test_minimal_ns,py.py
Fix file name

**Comment by Byon2** [comment]
Okay, fixed the issues with "commited 2". Do I need to open a new pull request for it or?

**Review by kasgel — APPROVED** [review]

File: src/tests/test_dns_recursion.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from unitest import TestCase
+from src.checks.dns_test_recursion import run
+
+class Test(TestCase):
+    def test_run:
+        # Here we check that RA gives false
+        res = run("google.com", ["ns1.google.com", "ns2.google.com", "ns3.google.com"])
+        assert res["result"]
```
Comment (kasgel) [inline src/tests/test_dns_recursion.py:8]: This is asserting that it returns true. Conflict with comment. Should it be `assert not res["result"]` ?

Comment (Byon2) [reply src/tests/test_dns_recursion.py:8]: Okay, I fixed it for both test programs and correct the comment for them

**Comment by kasgel** [comment]
See also this bug from yesterday: https://github.com/dnshealth/dns-health/issues/18

**Comment by Byon2** [comment]
.

**Comment by kasgel** [comment]
Also remove unnecessary files from inside the src/ folder since all your checks are already in src/checks?

**Comment by Byon2** [comment]
I think it looks good now

**Pull request merged**